src/sites_toolbox.py

```diff
@@ -28,6 +30,36 @@ def get_user_details(session, email):
     ]
     return user_sites, user_site_group, user_site_count
 
+# get details for one site
+
+def get_site_details(session, site_selection):
+  """Get the site details for one site"""
+  site_uuid = get_site_by_uuid(session=session, site_uuid=site_selection)
```

Comment:
should this be `site=get_site_by_uuid(session=session, site_uuid=site_selection)`